observability/opentelemetry/v2/client/cloudevents_carrier.go

```diff
@@ -0,0 +1,85 @@
+/*
+ Copyright 2021 The CloudEvents Authors
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package client
+
+import (
+	"context"
+
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"github.com/cloudevents/sdk-go/v2/extensions"
+)
+
+// CloudEventCarrier adapts the distributed trace extension to satisfy the TextMapCarrier interface.
+// https://github.com/open-telemetry/opentelemetry-go/blob/main/propagation/propagation.go#L23
```

Comment:
```suggestion
https://github.com/open-telemetry/opentelemetry-go/blob/v1.0.0-RC3/propagation/propagation.go#L23
```
Otherwise the line reference might become wrong (you can either select a branch from the dropdown on top or press `y` to have the branch replaced by the commit hash but I find the branch reference more expressive).

Reply:
Good point. I changed it here in other places as well.